### PR TITLE
Add nm2, μm2 (area) and nm (length) units

### PR DESCRIPTION
**Pull request by Aleaxandre**

### CODE REVIEW

File: src/definitions/__tests__/length.test.ts [hunk 1]
```diff
@@ -168,3 +168,31 @@ test('μm to m', () => {
...
+  });
+  expect(convert(1).from('m').to('nm')).toBeCloseTo(1e9);
+});
+
+test('nm to m', () => {
+  const convert = configureMeasurements<'length', LengthSystems, LengthUnits>({
+    length,
+  });
+  expect(convert(1).from('nm').to('m')).toBe(1e-9);
+});
```
Comment (Taar) [inline src/definitions/__tests__/length.test.ts:198]: This test case already exists
```suggestion
```

File: src/definitions/__tests__/length.test.ts [hunk 2]
```diff
@@ -168,3 +168,31 @@ test('μm to m', () => {
...
+  });
+  expect(convert(1).from('m').to('μm')).toEqual(1e6);
+});
+
+test('µm to m', () => {
+  const convert = configureMeasurements<'length', LengthSystems, LengthUnits>({
+    length,
+  });
+  expect(convert(1).from('μm').to('m')).toBe(1e-6);
+});
```
Comment (Taar) [inline src/definitions/__tests__/length.test.ts:184]: This test case already exists
```suggestion
```

**Review by Taar — CHANGES_REQUESTED** [review]
The area units look good but I'm confused by the PR's title. `nm` is already a unit of `length`. So was the goal to add in additional unit tests for `nm`?

**Comment by Aleaxandre** [comment]
Sorry, my mistake, I updated the pull request the way you suggested.
Thanks !

**Review by Taar — APPROVED** [review]

**Comment by cyrinebensalem** [comment]
hello @Taar  can we have a release including this update? thank you

**Comment by Taar** [comment]
> hello @Taar can we have a release including this update? thank you

https://www.npmjs.com/package/convert-units/v/3.0.0-beta.3

**Pull request merged**